extension/src/test/fixtures/expShow/sorted/rows.ts

```diff
@@ -0,0 +1,348 @@
+import { join } from '../../../util/path'
```

Comment:
<img width="233" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/a2bb1f0d-3e11-4305-a271-294306afbd2c">

The large numbers are mainly due to:
* adding a new code fixture for Storybook (`fixtures/expShow/sorted`)
* Moving the `getRowModel` mock function value outside of a describe block inside of `TableContent.tsx`

Reply:
Probably should have moved the storybook changes in a separate chain pr 😅